apps/ui/src/components/SpaceTreasury.vue

```diff
@@ -378,16 +378,18 @@ watchEffect(() => setTitle(`Treasury - ${props.space.name}`));
             v-if="nftsLoading && !nftsLoaded"
             class="px-4 py-3 block"
           />
-          <div class="flex flex-row flex-wrap gap-4 p-4">
+          <div
+            class="grid grid-cols-1 minimum:grid-cols-2 sm:grid-cols-3 md:grid-cols-4 xl:grid-cols-5 2xl:grid-cols-7 3xl:grid-cols-9 gap-4 gap-y-2 max-w-fit mx-auto p-4"
+          >
             <a
               v-for="(nft, i) in nfts"
               :key="i"
-              :href="sanitizeUrl(nft.permalink) || '#'"
+              :href="sanitizeUrl(nft.opensea_url) || '#'"
               target="_blank"
-              class="block max-w-[120px]"
+              class="block w-[120px] shrink-0"
```

Comment:
Currently, on most phones, we end up with only 2 cols, in the middle

![Screenshot 2024-09-14 at 23 44 05](https://github.com/user-attachments/assets/39ff60c3-2946-4aa0-8fb8-97f17c2de9bd)

@bonustrack What do you think of using 100% width for the card content on small screen (`w-[100%] sm:w-[120px]`), so it looks less empty, or is it too big ?

![Screenshot 2024-09-14 at 23 44 32](https://github.com/user-attachments/assets/8fe3d668-fa0b-4bc3-b810-62eda84ccf09)

Reply:
@bonustrack could we increase the width on smaller screens??

Reply:
@ChaituVR Yes we can do that, looks better